src/components/Card.js

```diff
@@ -9,9 +9,9 @@ const Card = (props) => {
   return (
     <div className="card">
       <section className="card_content">
-        <p className="card__content-text"> {props.cardText} </p>
+        <p className="card__content-text"> {props.text} </p>
```

Comment:
cardText and cardEmoji is right, but wasn't passed in using the same name in the prototypes in this file. In Board.js, initially, we were using just text and emoji, so now we are just using text and emoji instead so it all matches.